events.html

```diff
@@ -60,5 +60,6 @@ <h4>To find out about more events outside this month, checkout the offical USACS
         <a href="https://usacs.slack.com/"><img src="assets/img/slack-mark.png"></a>
       </div>
     </footer>
+    <script src="assets/js/footer.js"></script>
```

Comment:
Shouldn't this be in a template somewhere?

Reply:
It is in the template, that why it got compiled in

Reply:
Oh yeah... I'll just have to ctrl+f "mustache" for all the template things.